Todo.WebApi/Controllers/TodoController.cs

```diff
@@ -47,11 +67,11 @@ public IActionResult Add([FromBody]TodoTask task)
         public IActionResult Remove([FromBody]Guid id)
         {
             var list = _todoLists.GetPublic();
-            if(list is null) { return NotFound("The public list doesn't exist."); }
+            if (list is null) { return NotFound("The public list doesn't exist."); }
 
             var taskToDelete = list.Tasks.FirstOrDefault(t => t.Id == id);
```

Comment:
`var taskToDelete = list.Tasks.Find(t => t.Id == id);`
This seems more logical to me (returns null if none found).

Reply:
We're using an ICollection for the tasks which doesn't have any method for `.Find()` thus we went with `FirstOrDefault()`